### PR TITLE
My6sense new adapter

### PR DESCRIPTION
**Pull request by my6sense**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [X] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

First push for new adapter

- test parameters for validating bids
```
{
  bidder: 'my6sense',
  params: {    
    key: 'mW3mLht0kMT',
    pageUrl: '[PAGE_URL]',
    zone: '[ZONE]',
    dataView: ''
    organicClicks:'',
    paidClicks:''
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by bretg** [comment]
Docs PR is https://github.com/prebid/prebid.github.io/pull/846

File: modules/my6senseBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
+const {registerBidder} = require('../src/adapters/bidderFactory');
+const BIDDER_CODE = 'my6sense';
+const END_POINT = 'http://papi.mynativeplatform.com/pub2/web/hbwidget.json';
```
Comment (mike-chowla) [inline modules/my6senseBidAdapter.js]: Adapter does not support https which is required

**Comment by my6sense** [comment]
@mike-chowla - we added a version number to the endpoint url and removed the "http" from the url it self and now it starts with // to make sure we support ssl as well.

**Comment by my6sense** [comment]
@mike-chowla, we made the changes and pushed them a week ago.
is there anything else we need to do?

**Comment by mike-chowla** [comment]
@my6sense Apologies for the delay.  

Question:  I don't see your adapter declaring supportedMediaTypes which I believe means you'll only be bidding manner.  I looked at your doc PR which list supported media type as native.

Are you intending to bid on banner, banner & native, or just native?

**Comment by my6sense** [comment]
@mike-chowla, i just added the "supportedMediaTypes" property to the spec object, we are using BANNER & NATIVE types, thanks.

**Comment by mike-chowla** [comment]
I'm not getting bids back due to a CORS error.

screen shot 2018-07-09 at 11 34 16 am

**Comment by my6sense** [comment]
@mike-chowla, i updated our my6senseBidAdapter.md file with a valid widget key "OAJJBW2LRYi2CxfhzqogkA", please re-check it.
thanks

**Pull request merged**